Frends.Community.SQL/Frends.Community.SQL.csproj

```diff
@@ -9,7 +9,7 @@
     <IncludeSource>true</IncludeSource>
     <PackageTags>Frends</PackageTags>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <Version>1.1.3</Version>
+    <Version>1.1.4</Version>
```

Comment:
You're adding a whole new Task, so the minor version should be increased instead of build version.

Reply:
Bumped to version 1.2.4.